src/parser/warrior/fury/modules/spells/MissedRampage.js

```diff
@@ -2,57 +2,88 @@ import React from 'react';
 
 import Analyzer from 'parser/core/Analyzer';
 
+import Events from 'parser/core/Events';
+
 import SPELLS from 'common/SPELLS';
 import RESOURCE_TYPES from 'game/RESOURCE_TYPES';
+import { SELECTED_PLAYER } from 'parser/core/EventFilter';
 
 import SpellLink from 'common/SpellLink';
 
-const GENERATORS = [
-  SPELLS.RAGING_BLOW.id,
-  SPELLS.BLOODTHIRST.id,
-  SPELLS.EXECUTE.id,
+/*  Example log:
+ *  https://www.warcraftlogs.com/reports/KhynM7v96cZkTBdg#fight=6&type=damage-done&source=78
+ */
+
+const rageGenerators = [
```

Comment:
i believe the standard for constants declared up top is to use all caps, so RAGE_GENERATORS instead of rageGenerators. Not sure if that is a hard rule or not, just what ive seen in all the modules ive looked at.